### PR TITLE
Adding podman

### LINKED ISSUE #15: Add RH tooling:  Podman
Add Podman to EC2 instances

### PR DESCRIPTION
**Pull request by itroyano**
Fixes #15

### CODE REVIEW

File: roles/ec2userdata/tasks/main.yml [hunk 1]
```diff
@@ -121,6 +121,12 @@
     dest: /usr/local/bin
     remote_src: true
 
+- name: Install Podman
```
Comment (bcrochet) [inline roles/ec2userdata/tasks/main.yml]: Since this is packaged, this should just be added to the list at L15.

**Review by bcrochet — CHANGES_REQUESTED** [review]
Move to existing package task.

File: roles/ec2userdata/tasks/main.yml (hunk 1 above)
Comment (itroyano) [reply roles/ec2userdata/tasks/main.yml]: Makes sense.  done.

**Comment by itroyano** [comment]
> Move to existing package task.

Done.  thanks @bcrochet !

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**